storage-manager/src/Ownership.cpp

```diff
@@ -124,28 +138,24 @@ bf::path Ownership::get(const bf::path& p, bool getOwnership)
   return ret;
 }
 
-// minor timesaver
-#define TOUCH(p, f)                                                                                       \
-  {                                                                                                       \
-    int fd = ::open((metadataPrefix / p / f).string().c_str(), O_TRUNC | O_CREAT | O_WRONLY, 0660);       \
-    if (fd >= 0)                                                                                          \
-      ::close(fd);                                                                                        \
-    else                                                                                                  \
-    {                                                                                                     \
-      char buf[80];                                                                                       \
-      int saved_errno = errno;                                                                            \
-      cerr << "failed to touch " << metadataPrefix / p / f << " got " << strerror_r(saved_errno, buf, 80) \
-           << endl;                                                                                       \
-    }                                                                                                     \
-  }
-
-#define DELETE(p, f) ::unlink((metadataPrefix / p / f).string().c_str());
-
 void Ownership::touchFlushing(const bf::path& prefix, volatile bool* doneFlushing) const
 {
   while (!*doneFlushing)
   {
-    TOUCH(prefix, "FLUSHING");
+    {
+      auto kvStorage = KVStorageInitializer::getStorageInstance();
+      auto tnx = kvStorage->createTransaction();
+      const std::string flushingKey = KVPrefixes[static_cast<uint32_t>(KVPrefixId::SM_OWNERSHIP)] +
```

Comment:
TBH it is very hard to read which idx is used in the end.
I suggest to put idx-es cast outside to improve readability. Moreover I suggest to replace the key creation pattern  with an inline func that leverages ostringstream with preallocated buffer. WDYT @denis0x0D ?

Reply:
make sense, it could be placed in a outline function.